src/glb-director/glb_director_config.c

```diff
@@ -321,6 +323,53 @@ glb_director_config *glb_director_config_load_file(const char *config_file,
 		}
 	}
 
+	/* Hash field specification.
+	 * By default, we include a single hash of source IP.
+	 */
+	cfg->hash_fields.src_addr = 1;
+	cfg->hash_fields.dst_addr = 0;
+	cfg->hash_fields.src_port = 0;
+	cfg->hash_fields.dst_port = 0;
+	cfg->use_alt_hash_fields = 0;
+
+	json_t *hash_fields = json_object_get(root, "hash_fields");
+	if (hash_fields != NULL) {
+		if (parse_hash_fields(&cfg->hash_fields, hash_fields) != 0) {
+			glb_log_error(
+			    "hash_fields must contain at least 1 field if specified");
```

Comment:
This log message (and the alt) below are making assumptions about why `parse_hash_fields` failed that could potentially change in the future. Probably best to move this error message to `parse_hash_fields` (and passing in the "hash_fields" or "alt_hash_fields" part) so that if parsing involves more i the future the actual error handling will work as-is and the error message can be tailored for circumstances.